sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/siprouting/_sip_routing_client.py

```diff
@@ -0,0 +1,251 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING  # pylint: disable=unused-import
+
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse  # type: ignore
+
+from azure.core.tracing.decorator import distributed_trace
+
+from._models import SipTrunk
+from ._generated.models import (
+    SipConfiguration,
+    SipTrunkRoute,
+    SipTrunkInternal
+)
+from ._generated._sip_routing_service import SIPRoutingService
+from .._shared.utils import (
+    parse_connection_str,
+    get_authentication_policy
+)
+from .._version import SDK_MONIKER
+
+if TYPE_CHECKING:
+    from typing import List, Any
+    from azure.core.credentials import TokenCredential
+
+
+class SipRoutingClient(object):
+    """A client to interact with the AzureCommunicationService SIP routing gateway.
+    This client provides operations to retrieve and manage SIP routing configuration.
+
+    :param endpoint: The endpoint url for Azure Communication Service resource.
+    :type endpoint: str
+    :param credential: The credentials with which to authenticate.
+    :type credential: TokenCredential
+    :keyword api_version: Api Version. Default value is "2021-05-01-preview". Note that overriding
+     this default value may result in unsupported behavior.
+    :paramtype api_version: str
+    """
+
+    def __init__(
+        self,
+        endpoint,  # type: str
+        credential,  # type: TokenCredential
+        api_version="2021-05-01-preview", # type: str
```

Comment:
This param shouldn't be positional - intead we can just `pop` it from kwargs.

Reply:
Removed as positional. The parameter is actually not used in this class, but further down the line is popped from the kwargs.